p4-14/v1.0.5/tex/p4.pt

```diff
@@ -1790,12 +1801,6 @@ If the \texttt{header_instance} is not an element in a header stack, the indicat
 header instance is set valid. If the header instance was invalid, all its 
 fields are initialized to 0. If the header instance is already valid, it is 
 not changed.
-
-If \texttt{header_instance} is an element in a header stack, the effect is to push 
-a new header into the stack at the indicated location. Any existing valid 
-instances from the given index or higher are copied to the next higher index. 
-The given instance is set to valid. If the array is fully populated when this 
-operation is executed, then no change is made to the Parsed Representation.
```

Comment:
The paragraph before this one only defines the behavior of add_header on a header that is not an element of a header stack.  I thought perhaps that the intent was to continue to allow add_header to be performed on a specified index of a header stack, but that its behavior would be specified to make that one element of the header stack become valid if it was invalid before, but there would be no effect on any other elements of the header stack.